src/pages/projects/[project]/measures/Measures.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+
+import ReactEcharts, { EChartsOption } from "echarts-for-react";
+
+import { Box, Container } from '@mui/material';
+
+import Layout from '@components/Layout';
+
+import formatMeasuresHistoryChartData from '@utils/formatMeasuresHistory';
+import ProjectContent from '../components/ProjectContent';
+import useQuery from '../components/ProjectContent/hook/useQuery';
+
+
+const Measures: NextPage = () => {
```

Comment:
Esse component não é uma Page
```suggestion
const Measures = () => {
```